process/smartContract/process.go

```diff
@@ -733,17 +708,18 @@ func (sc *scProcessor) saveSCOutputToCurrentState(output *vmcommon.VMOutput, rou
 	sc.mutSCState.Lock()
 	defer sc.mutSCState.Unlock()
 
-	if _, ok := sc.mapExecState[round]; !ok {
-		sc.mapExecState[round] = scExecutionState{
-			allLogs:       make(map[string][]*vmcommon.LogEntry),
-			allReturnData: make(map[string][]*big.Int),
-			returnCodes:   make(map[string]vmcommon.ReturnCode)}
-	}
+	/*
+		if _, ok := sc.mapExecState[round]; !ok {
```

Comment:
`mapExecState` has been mostly commented out, but it is still present as a field of the `scProcessor` struct. If it's not needed anymore, it should be removed.